### PR TITLE
fix: Made it so the visibility toggle won't flicker.

### PR DESCRIPTION
**Pull request by m6jones**
Made it so the visibility toggle won't flicker.

### CODE REVIEW

File: components/activity/editor/d2l-activity-editor-footer.js [hunk 1]
```diff
@@ -3,10 +3,9 @@ import '@brightspace-ui/core/components/backdrop/backdrop.js';
 import '@brightspace-ui/core/components/button/button.js';
 import '@brightspace-ui/core/components/dialog/dialog.js';
 import '../../common/d2l-activity-visibility.js';
-import { css, LitElement } from 'lit-element/lit-element.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
 import { HypermediaStateMixin, observableTypes } from '@brightspace-hmc/foundation-engine/framework/lit/HypermediaStateMixin.js';
 import { getUniqueId } from '@brightspace-ui/core/helpers/uniqueId.js';
-import { html } from '@brightspace-hmc/foundation-engine/framework/lit/hypermedia-components.js';
```
Comment (m6jones) [inline components/activity/editor/d2l-activity-editor-footer.js:9]: This component is currently not using any of the dynamic components so switching to html from lit.

File: components/common/d2l-activity-visibility.js [hunk 2]
```diff
@@ -47,20 +47,18 @@ class ActivityVisibilityEditorToggle extends HypermediaStateMixin(LitElement) {
...
 		}
-		return null;
+
+		return html`
+		<d2l-switch-visibility
+			?disabled="${!this.switchEnabled}"
+			@change="${this._onChange}"
+			?on="${!this._isDraft}"
+			text-position="${this._textHidden ? 'hidden' : 'end'}">
+		</d2l-switch-visibility>`;
```
Comment (m6jones) [inline components/common/d2l-activity-visibility.js:61]: Inverted the if to make it more "clean"

File: components/activity/editor/d2l-activity-editor-footer.js (hunk 1 above)
Comment (m6jones) [reply components/activity/editor/d2l-activity-editor-footer.js:9]: https://github.com/BrightspaceHypermediaComponents/foundation-engine/issues/79

**Review by tenbradley — APPROVED** [review]
Looks good. Tested on local bsi.

**Review by AKobets — APPROVED** [review]

**Pull request merged**